slimCat/Views/UI Parts/ChannelMessageView.xaml

```diff
@@ -35,7 +36,8 @@
 
     <InlineUIContainer BaselineAlignment="Bottom">
         <Rectangle Height="{Binding Source={x:Static models:ApplicationSettings.FontSize}}" Width="{Binding Source={x:Static models:ApplicationSettings.FontSize}}"
-                   Fill="{Binding Path=Poster, Converter={StaticResource GenderColorConverter}, Mode=OneWay}">
+                   Fill="{Binding Path=Poster, Converter={StaticResource GenderColorConverter}, Mode=OneWay}"
+                   Visibility="{Binding Source={x:Static models:ApplicationSettings.ShowGenderIcons}, Converter={StaticResource GreaterThanZeroConverter}}">
```

Comment:
`GreaterThanZeroConverter` is not a typo? It seems like you want `BoolConverter` here